### PR TITLE
feat: Add test for deprecation warning

### PR DESCRIPTION
**Pull request by dipinknair**
add additional case for deprecation warning

### CODE REVIEW

File: tests/embedding/test_app.py [hunk 1]
```diff
@@ -47,6 +47,9 @@ def test_deprecation_warning(embedded_app):
     struct = embedded_app.Model.AddStaticStructuralAnalysis()
     with pytest.warns(UserWarning):
         struct.SystemID
+    harmonic_acoustic = embedded_app.Model.AddHarmonicAcousticAnalysis()
+    with pytest.warns(UserWarning):
```
Comment (koubaa) [inline tests/embedding/test_app.py:51]: minor - you can get SystemID from a harmonic acoustic.  So you don't really need a static structural anymore

**Review by koubaa — APPROVED** [review]

File: tests/embedding/test_app.py (hunk 1 above)
Comment (dipinknair) [reply tests/embedding/test_app.py:48]: Done

File: tests/embedding/test_app.py [hunk 2]
```diff
@@ -44,9 +44,10 @@ def test_app_repr(embedded_app):
 @pytest.mark.embedding
 @pytest.mark.minimum_version(241)
 def test_deprecation_warning(embedded_app):
-    struct = embedded_app.Model.AddStaticStructuralAnalysis()
+    harmonic_acoustic = embedded_app.Model.AddHarmonicAcousticAnalysis()
     with pytest.warns(UserWarning):
-        struct.SystemID
+        harmonic_acoustic.SystemID
+        harmonic_acoustic.AnalysisSettings.MultipleRPMs = True
```
Comment (koubaa) [inline tests/embedding/test_app.py:50]: minor - putting both in a `pytest.warns` will technically allow the test to pass if one of these does not cause a deprecation warning. A better design would be to put each in its own `pytest.warns` block

Comment (dipinknair) [reply tests/embedding/test_app.py:50]: I already merged this. But I will make these changes in the other PR

**Pull request merged**